gcloud/monitoring/agent_translation_service_api.py

```diff
@@ -0,0 +1,143 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# EDITING INSTRUCTIONS
+# This file was generated from the file
+# https://github.com/google/googleapis/blob/master/google/monitoring/v3/agent_service.proto,
+# and updates to that file get reflected here through a refresh process.
+# For the short term, the refresh process will only be runnable by Google engineers.
+# Manual additions are allowed because the refresh process performs
+# a 3-way merge in order to preserve those manual additions. In order to not
+# break the refresh process, only certain types of modifications are
+# allowed.
+#
+# Allowed modifications:
+# 1. New methods (these should be added to the end of the class)
+#
+# Happy editing!
+
+from google.gax import PageDescriptor
+from google.gax import api_callable
+from google.gax import config
+from google.gax.path_template import PathTemplate
+from google.monitoring.v3 import agent_service_pb2
+from google.api import monitored_resource_pb2
+from google.monitoring.v3 import agent_pb2
+
+
+class AgentTranslationServiceApi(object):
+    """
+    The AgentTranslation API allows `collectd`-based agents to
+    write time series data to Cloud Monitoring.
+    See [google.monitoring.v3.MetricService.CreateTimeSeries] instead.
+    """
+
+    # The default address of the logging service.
+    _SERVICE_ADDRESS = 'monitoring.googleapis.com'
+
+    # The default port of the logging service.
+    _DEFAULT_SERVICE_PORT = 443
+
+    # The scopes needed to make gRPC calls to all of the methods defined in
+    # this service
+    _ALL_SCOPES = (
+        'https://www.googleapis.com/auth/monitoring',
+        'https://www.googleapis.com/auth/monitoring.write',
+        'https://www.googleapis.com/auth/monitoring.read',
+        'https://www.googleapis.com/auth/cloud-platform',
+    )
+
+
+
+    class Templates(object):
+        """PathTemplates for resources used by AgentTranslationServiceApi."""
+        PROJECT = PathTemplate.from_string(
+            'projects/{project}')
```

Comment:
This should be defined at module-level, though it's unclear why it is needed at all.